include/image.h

```diff
@@ -363,6 +365,12 @@ static void __attribute__((noinline)) wolfBoot_image_clear_signature_ok(
     asm volatile("cmp r2, r0":::"cc"); \
     asm volatile("bne .-12")
 
+/* Some SHA checks */
+#if !defined(WOLFBOOT_SHA_DIGEST_SIZE) || (WOLFBOOT_SHA_DIGEST_SIZE <= 0)
+#   error "WOLFBOOT_SHA_DIGEST_SIZE must be defined"
```

Comment:
This is actually redundant. The check is performed in wolfboot.h . See error at wolfboot.h:225